packages/slate-types/src/lib/isAlignableElement.ts

```diff
@@ -1,8 +1,15 @@
 import type { HeadingNode, ListNode, ParagraphNode, QuoteNode } from '../nodes';
+import { CalloutNode } from '../nodes';
 import { isHeadingNode, isListNode, isParagraphNode, isQuoteNode } from '../nodes';
 
 export type AlignableNode = HeadingNode | ListNode | QuoteNode | ParagraphNode;
 
 export function isAlignableElement(node: any): node is AlignableNode {
-    return isParagraphNode(node) || isHeadingNode(node) || isQuoteNode(node) || isListNode(node);
+    return (
+        isParagraphNode(node) ||
+        isHeadingNode(node) ||
+        isQuoteNode(node) ||
+        isListNode(node) ||
+        CalloutNode.isCalloutNode(node)
```

Comment:
_Comment in an unrelated place_

Btw, I believe we should also add support for this new node in Automatic Story Translation lambdas as well.

https://github.com/prezly/prezly-auto-translation-lambdas/blob/46e8274c0164ed33fec5baacc3e51119a8bf92c0/src/lib/serialize/serializeToXml.ts#L28-L32

Reply:
Good remark. Thanks! :+1:

Reply:
DEV-13045

Reply:
https://github.com/prezly/prezly-auto-translation-lambdas/pull/10